requirements.txt

```diff
@@ -1,3 +1,4 @@
 # the default package dependencies
 -r ./requirements/app/base.txt
+-r ./requirements/lite/base.txt
```

Comment:
Why add this here if pytorch/base.txt includes it?

Reply:
```suggestion
```
good point :)

Reply:
true...

Reply:
```suggestion
```